scintilla/gtk/ScintillaGTK.cxx

```diff
@@ -3250,6 +3250,7 @@ void scintilla_release_resources(void) {
 static void *copy_(void *src) { return src; }
 static void free_(void *doc) { }
 
+GEANY_API_SYMBOL
 GType scnotification_get_type(void) {
 	static gsize type_id = 0;
```

Comment:
> It's kinda odd g_once_init_*() API claims a void* pointer though, so I don't know.

Yeah, and I think even `gsize` isn't the correct type, it's probably meant to be `guintptr`, but oh well.

> Anyway the discussion out of scope for this PR

Code review is out of scope for this PR? At the least it could do like the code @b4n linked to, and avoid ugly (and useless) casts.

> and really minor anyway

This is true